_topic_maps/_topic_map.yml

```
@@ -2992,6 +2992,8 @@ Topics:
   File: ztp-precaching-tool
 - Name: Image-based upgrade for single-node OpenShift clusters
   File: ztp-image-based-upgrade
+- Name: Image-based installation for single-node OpenShift clusters
+  File: ibi-image-based-install
```

Maybe: 

```suggestion
- Name: Image-based installation for single-node OpenShift clusters
  File: ibi-image-based-install
- Name: Factory installing single-node OpenShift with image-based installation 
  File: ibi-image-based-factory-install
```

This keeps the common conceptual stuff separate